tests/phpunit/Compat/WC_Stripe_Subscriptions_Helper_Test.php

```
@@ -335,4 +359,72 @@ public function test_get_detached_payment_data_from_subscription() {
+				'expected'     => false,
+			],
+			'HPOS disabled, subscriptions edit page'     => [
+				'HPOS enabled' => false,
+				'query params' => [
+					'post'   => $subscription,
+					'action' => 'edit',
+				],
+				'expected'     => true,
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provide_test_is_subscription_edit_page is not named in camelCase. (undefined)

(CamelCaseMethodName)